files/zh-tw/web/javascript/reference/global_objects/function/bind/index.md

```diff
@@ -224,7 +224,7 @@ if (!Function.prototype.bind) {
 Some of the many differences (there may well be others, as this list does not seriously attempt to be exhaustive) between this algorithm and the specified algorithm are:
 
 - The partial implementation relies on {{jsxref("Array.prototype.slice()")}}, {{jsxref("Array.prototype.concat()")}}, {{jsxref("Function.prototype.call()")}} and {{jsxref("Function.prototype.apply()")}}, built-in methods to have their original values.
-- The partial implementation creates functions that do not have immutable "poison pill" {{jsxref("Function.caller", "caller")}} and `arguments` properties that throw a {{jsxref("Global_Objects/TypeError", "TypeError")}} upon get, set, or deletion. (This could be added if the implementation supports {{jsxref("Object.defineProperty")}}, or partially implemented \[without throw-on-delete behavior] if the implementation supports the {{jsxref("Object.defineGetter", "__defineGetter__")}} and {{jsxref("Object.defineSetter", "__defineSetter__")}} extensions.)
+- The partial implementation creates functions that do not have immutable "poison pill" {{jsxref("Function.caller", "caller")}} and `arguments` properties that throw a {{jsxref("Global_Objects/TypeError", "TypeError")}} upon get, set, or deletion. (This could be added if the implementation supports {{jsxref("Object.defineProperty")}}, or partially implemented \[without throw-on-delete behavior] if the implementation supports the {{jsxref("Object.defineGetter", "**defineGetter**")}} and [`Object.prototype.__defineSetter__()`](/zh-TW/docs/Web/JavaScript/Reference/Global_Objects/Object/__defineSetter__) extensions.)
```

Comment:
```suggestion
- The partial implementation creates functions that do not have immutable "poison pill" {{jsxref("Function.caller", "caller")}} and `arguments` properties that throw a {{jsxref("Global_Objects/TypeError", "TypeError")}} upon get, set, or deletion. (This could be added if the implementation supports {{jsxref("Object.defineProperty")}}, or partially implemented \[without throw-on-delete behavior] if the implementation supports the [`Object.prototype.__defineGetter__()`](/zh-TW/docs/Web/JavaScript/Reference/Global_Objects/Object/__defineGetter__) and [`Object.prototype.__defineSetter__()`](/zh-TW/docs/Web/JavaScript/Reference/Global_Objects/Object/__defineSetter__) extensions.)
```